chromium/rules.js

```diff
@@ -228,7 +239,7 @@ RuleSets.prototype = {
     var platform = ruletag["platform"]
     if (platform) {
       default_state = false;
-      if (platform == "mixedcontent" && background.enableMixedRulesets) {
+      if (platform == "mixedcontent" && enableMixedRulesets) {
```

Comment:
Actually this check here and in other functions won't work because other JS files only modify property on `exports` object but local variable will remain intact. It's better to remove variable for `enableMixedRulesets` and always check `exports.enableMixedRulesets` instead (note that `domainBlacklist` is fine because it's only ever modified through its own methods and not replaced with another value like boolean).

Reply:
Good catch! I moved `enableMixedRulesets` and `domainBlacklist` into `rules.settings.*`. So now we refer to `settings.enableMixedRulesests in this file, and in other files, `rules.settings.enableMixedRulesets`. 

I did this because I think direct references to  `exports` should be reserved for actually exporting things.

Reply:
I don't think `settings` is the best name. These are really just stateful objects.